src/Entities/Entity.cpp

```diff
@@ -1589,25 +1590,46 @@ bool cEntity::MoveToWorld(cWorld * a_World, Vector3d a_NewPosition, bool a_SetPo
 		return false;
 	}
 
+	if (m_WorldChangeInfo.m_NewWorld != nullptr)
+	{
+		// Avoid scheduling multiple warp tasks
+		return true;
```

Comment:
The whole point of writing it this way was to take the _latest_ call, not the first call.

Reply:
Thanks, oops :P